integration-tests/api/src/services/local-file-service.js

```diff
@@ -66,4 +66,12 @@ export default class LocalFileService extends AbstractFileService {
       fileKey,
     }
   }
+
+  async getDownloadStream(fileData) {
+    const dir = (this.upload_dir_ =
+      process.env.UPLOAD_DIR ?? options.upload_dir ?? "uploads/images")
+
+    const filePath = path.resolve(dir, fileData.fileKey)
```

Comment:
suggestion: you xan use this.upload_dir_ directly here since it has been assign in the constructor.